src/babel-serializer/file.cc

```diff
@@ -2,7 +2,20 @@
 
 #include <fstream>
 
+#if defined(CC_OS_WINDOWS)
+#include <clean-core/native/win32_sanitized.hh>
+
+#include <WinBase.h>
+#include <fileapi.h>
```

Comment:
Not sure if it applies here but if you need more than Windows.h and still want to use the cc sanitized version, all of the required headers should instead be wrapped with the sanitize_before and sanitize_after inl files, for example how it's done here:
https://github.com/project-arcana/phantasm-hardware-interface/blob/kw/develop/src/phantasm-hardware-interface/d3d12/common/d3d12_sanitized.hh#L14
Otherwise the win32 headers can break internally

Reply:
I updated the the include section accordingly, assuming there is no need to check for the include guards like in your example, since this is a source file, not a header.